hal/src/rtl872x/sleep_hal.cpp

```diff
@@ -120,6 +169,16 @@ class SleepClass {
             }
         }
 
+        hal_interrupt_suspend();
+
+        /* We neeed to configure the IO expander interrupt before disabling SPI interface. */
+#if HAL_PLATFORM_IO_EXTENSION && MODULE_FUNCTION != MOD_FUNC_BOOTLOADER
+#if HAL_PLATFORM_MCP23S17
+        Mcp23s17::getInstance().interruptsSuspend();
```

Comment:
danger! - no corresponding call to `Mcp23s17::interruptsRestore`

Reply:
Good catch!